src/aphroditeInterfaceFactory.js

```diff
@@ -1,144 +1,18 @@
-  // method makes an effort to flip CSS styles for an RTL context.
-  // This function returns an object to be spread onto an element.
   resolve(styles) {
-    const flattenedStyles = flatten(styles);
-
-    const {
-      aphroditeStyles,
-      hasInlineStyles,
-      inlineStyles,
-    } = separateStyles(flattenedStyles);
-
-    const stylesWithDirection = aphroditeStyles.map((stylesObj) => {
-      let definition = stylesObj._definition;
-      const rtlStyles = generateRTLStyles(definition);
-
-      if (rtlStyles) {
-        // This applies the rtlStyles whenever dir="rtl" is set. This is because
-        // the interface knows nothing about the current directional context.
-        definition = { ...definition, [RTL_SELECTOR]: rtlStyles };
-      }
-
-      return {
-        ...stylesObj,
-        _definition: definition,
-      };
-    });
-
-    const result = {};
-    if (hasInlineStyles) {
-      const inlineRTLStyles = generateRTLStyles(inlineStyles);
-
-      if (inlineRTLStyles) {
-        // Because we know nothing about the current directional context, there
-        // is no way to determine whether or not the inline styles should be
-        // flipped. As a result, if there are inline styles to be flipped, we
-        // do so by converting them to classnames and providing styles in both
-        // an RTL and an LTR context.
-        // This may not work for all situations! For instance, when using inline
-        // styles to animate a component with react-motion or animated.js,
-        // converting to classes is likely to be way too slow. For those and
-        // other edge-cases, consumers should rely on the resolveNoRTL method
-        // instead.
-        const stylesDef = Object.assign(inlineStyles, { [RTL_SELECTOR]: inlineRTLStyles });
-        stylesWithDirection.push({
-          _name: `inlineStyles_${hashObject(stylesDef)}`,
-          _definition: stylesDef,
-        });
-      } else {
-        result.style = inlineStyles;
-      }
-    }
-
-    if (stylesWithDirection.length > 0) {
-      result.className = css(...stylesWithDirection);
-    }
-
-    return result;
+    return resolve(css, styles);
   },
 
-  // Styles is an array of properties returned by `create()`, a POJO, or an
-  // array thereof. POJOs are treated as inline styles. This version of the
-  // resolve function explicitly does no work to flip styles for an RTL context.
-  // This function returns an object to be spread onto an element.
   resolveNoRTL(styles) {
-    const flattenedStyles = flatten(styles);
-
-    const {
-      aphroditeStyles,
-      hasInlineStyles,
-      inlineStyles,
-    } = separateStyles(flattenedStyles);
-
-    const result = {};
-    if (aphroditeStyles.length > 0) {
-      result.className = css(...aphroditeStyles);
-    }
-
-    if (hasInlineStyles) {
-      result.style = inlineStyles;
-    }
-
-    return result;
+    return resolve(css, styles);
```

Comment:
why is this return value the same as what's returned in the `resolve` method above?

Reply:
It's not strictly necessary but it's just to be explicit that the resolveNoRTL is the same as the resolve in this interface implementation